README.md

```diff
@@ -34,7 +34,7 @@ Guidelines:
 | [WeChatQRCode](./models/qrcode_wechatqrcode)            | QR Code Detection and Parsing | 100x100    | 7.04           | 37.68        | ---             | ---          | ---         |
 | [DaSiamRPN](./models/object_tracking_dasiamrpn)         | Object Tracking               | 1280x720   | 36.15          | 705.48       | 76.82           | ---          | ---         |
 | [YoutuReID](./models/person_reid_youtureid)             | Person Re-Identification      | 128x256    | 35.81          | 521.98       | 90.07           | 44.61        | ---         |
-| [MP-PalmDet](./models/palm_detection_mediapipe)         | Palm Detection                | 256x256    | 15.57          | 168.37       | 50.64           | 62.45        | ---         |
+| [MP-PalmDet](./models/palm_detection_mediapipe)         | Palm Detection                | 192x192    | 11.09          | 63.79        | 83.20           | 33.81        | ---         |
```

Comment:
Slower on Jetson?

Reply:
After verification, this problem is caused by OpenCV version iteration. In version `4.6.0`, time cost on cuda should change **from 50ms to 30ms**. But in version `4.7.0`, time cost on cuda change **from 200ms to 80ms**.

This means that after the OpenCV version update, inference time of this palm detecion model is **4 times slower** on cuda. Let's solved this problem firstly.

Reply:
Leave as-is and will be fixed in the next release.

Reply:
This problem is fixed by PR https://github.com/opencv/opencv/pull/23243.